CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#7658](https://github.com/thanos-io/thanos/pull/7658) Store: Fix panic because too small buffer in pool.
 - [#7643](https://github.com/thanos-io/thanos/pull/7643) Receive: fix thanos_receive_write_{timeseries,samples} stats
 - [#7644](https://github.com/thanos-io/thanos/pull/7644) fix(ui): add null check to find overlapping blocks logic
+- [#7814](https://github.com/thanos-io/thanos/pull/7814) Store: label_values: fetch less postings.
```

Comment:
Last comment. Can we be more specific about the change? `fetch less postings` is vague.